app/policies/roles/institution/health_organization.json.erb

```diff
@@ -0,0 +1,51 @@
+[
+  {
+    "name": "Institution <%= institution_name %> Admin",
+    "policy": {
+      "statement": [
+        {
+          "resource": "institution/<%= institution_id %>",
+          "action": [
+            "institution:read",
+            "institution:update",
+            "institution:delete",
+            "institution:registerDevice",
+            "institution:createSite",
+            "institution:createEncounter",
+            "institution:createRole"
+          ],
+          "delegable": true
+        },
+        {
+          "resource": [
+            "site?institution=<%= institution_id %>",
+            "device?institution=<%= institution_id %>",
+            "testResult?institution=<%= institution_id %>",
+            "encounter?institution=<%= institution_id %>"
+          ],
+          "action": "*",
+          "delegable": true
+        },
+        {
+          "resource": "role?institution=<%= institution_id %>",
+          "action": "*",
+          "delegable": true
+        }
+      ]
+    }
+  },
+  {
+    "name": "Institution <%= institution_name %> Reader",
+    "policy": {
+      "statement": [
+        {
+          "resource": "institution/<%= institution_id %>",
+          "action": [
+            "institution:read"
+          ],
```

Comment:
Reader should have permission over the same resources (site, device, etc) as the admin, but without any create/update/delete actions